### DISCUSSION TITLE
Is there a formal way to request a new version?

### DISCUSSION THREAD

**Discussion by kindoflew** [github-linguist/linguist#6154] [Q&A] [answered]
Basically the title -- is there a formal (or preferred) way to request that a new release be published to incorporate changes from an upstream grammar? Thanks!

**Comment by lildude** [comment, marked as answer]
Nope. It happens when I get the chance depending on my workload and GitHub.com deploy locks etc. My intention is to get a new release out before each major GitHub Enterprise Server (GHES) release which happens approximately every quarter so the latest Server release can get the latest and greatest at the time of code freeze.

Due to vacation and [GitHub Universe](https://githubuniverse.com/), I won't be making the next release until the week of 14 Nov at the very earliest. This is also the last chance for me to get the changes in before the next GHES release so it's pretty much a sure thing 😁.

**Comment by kindoflew** [comment]
Thanks for the quick reply! And also for maintaining all this -- it's gotta be pretty wild working on a project that relies so much on so many *other* repos!
